src/main/resources/changesets/changelog-consent-51.0.xml

```diff
@@ -0,0 +1,13 @@
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet author="grushton" id="51.0">
+        <sql dbms="postgres">
+            alter table election alter column archived set default false;
+        </sql>
```

Comment:
This should have been covered in the `pgloader` migration, but isn't in the final result.